rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -50,6 +50,9 @@ struct PlayOptions
   // Sleep before play. Negative durations invalid. Will delay at the beginning of each loop.
   rclcpp::Duration delay = rclcpp::Duration(0, 0);
 
+  // Start paused.
+  bool paused = false;
```

Comment:
Is this boolean saying if we're _starting_ paused? Or if we're _currently_ paused? The comment says the first, but the variable name implies the other.

Reply:
It's saying we're starting paused.
I agree that the variable naming is confusing, fixed in https://github.com/ros2/rosbag2/pull/904/commits/4ff72610999249babfabf2b97ae1d1cb177182bf.